dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -225,27 +233,57 @@ def SignalHandler(*unused_argvs):
   sys.exit(0)
 
 
+def SetupLogging():
+  """Sets up a logging handler with dftimewolf's custom formatter."""
+  # Clear root handlers (Turbinia is setting them)
+  root_log = logging.getLogger()
+  root_log.handlers = []
+
+  # Add and silence a default stream handler, this is automatically set
```

Comment:
s/Add and silence a default stream handler/Add a silence default stream handler/

Reply:
Done.